lib/googmodule.js

```diff
@@ -24,14 +24,15 @@ var createPreprocesor = function(logger, /* config.basePath */ basePath) {
 
     log.debug('Processing "%s".', file.originalPath);
 
-    // Path must include http:// protocol and '/base/' so that Karma can still recognize the URL
-    // and apply source maps.
-    content =
-        JSON.stringify(content + '\n//# sourceURL=http://googmodule/base/' + relativePath + '\n');
-    // Make sure not to insert any line breaks before the content, so that line
-    // numbers match the original. It's a poor man's source map, essentially.
-    var output = '/* Generated from ' + relativePath + ' by karma-googmodule-preprocessor */ ' +
-                 'goog.loadModule(' + content + ');\n';
+    var output = `/* Generated from ${relativePath} by karma-googmodule-preprocessor */
+goog.loadModule(function(exports) {
+"use strict";
+${content}
+//# sourceURL=http://googmodule/base/${relativePath}
+return exports;
```

Comment:
Looks like this is just passed through? 

According to the doc for goog.loadModule():
```js
  //  in a eval forbidden environment (CSP) we allow a function definition
  // which in its body must call `goog.module`,
```
https://github.com/google/closure-library/blob/master/closure/goog/base.js#L1114

But there is no constraint here enforcing the call to `goog.module()`.

Reply:
Isn't the constraint the `MODULE_REGEXP` test on line 19?